spec/peep_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../models/peep'
+
+describe Peep do
+  it "can create peeps" do
+    subject = Peep.create({ description: "This is a test peep" })
```

Comment:
this can be written as : expect(Peep).to receive(:create).with()
To test the behaviour